### DISCUSSION TITLE
docfx random-generated filename ends with .csx

### DISCUSSION THREAD

**Discussion by zp-zpanda** [dotnet/docfx#8317] [Q&A] [answered]
I was just building my docs, and suddenly there are 1K+ errors in VS Code, which luckily don't interfere with my project. It seems that a random docfx-generated filename ends with .csx, which is the extension for interactive C# scripts. Is there a way to blacklist .csx to prevent the random errors from coming out of nowhere?

**Comment by KalleOlaviNiemitalo** [comment, marked as answer]
This would be fixed as part of <https://github.com/dotnet/docfx/issues/8203>. Until then, is it possible to make VS Code ignore the whole cache directory?

**Reply by zp-zpanda** [reply]
> is it possible to make VS Code ignore the whole cache directory?

It's possible. I've just tried it.

> This would be fixed as part of https://github.com/dotnet/docfx/issues/8203

Thanks a lot!
